calico-enterprise/operations/ebpf/enabling-ebpf.mdx

```diff
@@ -89,6 +90,8 @@ The best way to do that varies by Kubernetes distribution:
   `api-int.<cluster_name>.<base_domain>` should point to the API server or to the load balancer in front of the
   API server. Use that (filling in the `<cluster_name>` and `<base_domain>` as appropriate for your cluster) for the
   `KUBERNETES_SERVICE_HOST` below. Openshift uses 6443 for the `KUBERNETES_SERVICE_PORT`.
+- MKE runs a reverse proxy in each node, which can be used to reach the API server. You should use `proxy.local`
```

Comment:
Wait, didn't we have problems with pod-networked pods when using that?  Has that been fixed in operator?

Reply:
After setting ```bpfHostConntrackBypass``` to ```false```, I did not see any issue with pod networked pods coming up.